packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -222,4 +277,24 @@ extension on String {
     final file = File(path)..createSync(recursive: true);
     await response.pipe(file.openWrite());
   }
+
+  Future<void> downloadAndExtract(String path) async {
```

Comment:
Nit: named parameter would be useful here (eg. `downloadAndExtract({required String outputPath})` or similar)

Reply:
Good call, done 👍